central/integrationhealth/datastore/datastore.go

```diff
@@ -19,6 +19,7 @@ type DataStore interface {
 	GetRegistriesAndScanners(ctx context.Context) ([]*storage.IntegrationHealth, error)
 	GetNotifierPlugins(ctx context.Context) ([]*storage.IntegrationHealth, error)
 	GetBackupPlugins(ctx context.Context) ([]*storage.IntegrationHealth, error)
+	GetDeclarativeConfigs(ctx context.Context) ([]*storage.IntegrationHealth, error)
 
 	UpdateIntegrationHealth(ctx context.Context, integrationHealth *storage.IntegrationHealth) error
```

Comment:
Let's use the fact we are touching this part of code and rename to `UpsertIntegrationHealth` - either here or in a separate PR, whatever you prefer.